services/sonar/sonar-base.spec.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+
```

Comment:
At some point I thought I was going to need some traditional unit tests but no longer appears to be the case 🤔 

Will probably delete this as part of some final cleanup